WordPress/metadata/release_notes.txt

```diff
@@ -1,5 +1,10 @@
-- Added Site Menu and Home tabs to My Site screen
-- Made Add Block button more visible when block editor opens
-- Removed media/audio block upload error text
-- Added Site Intent screen to site creation process
-- Fixed crash caused by notification details for comments on GB File blocks
+* [*] App Settings: updated look and functionality of Locale Picker [https://github.com/wordpress-mobile/WordPress-Android/pull/16331]
+* [*] Fix issue that prevented Jetpack customers with standalone VideoPress subs uploading >5min videos [https://github.com/wordpress-mobile/WordPress-Android/pull/16322]
+* [*] On self-hosted sites, allow previewing unsaved changes to draft posts [https://github.com/wordpress-mobile/WordPress-Android/pull/16296]
+* [*] Editor: Fix issue that caused previews on atomic sites to sometimes be out-of-date. [https://github.com/wordpress-mobile/WordPress-Android/pull/16332]
+* [*] Fixes Media Picker video selection Accessibility issue [https://github.com/wordpress-mobile/WordPress-Android/pull/16426]
+* [*] Fixes self-hosted upload failure when the filename is non-latin [https://github.com/wordpress-mobile/WordPress-Android/pull/16421]
+* [*] Fixes self-hosted blurred featured image on post settings [https://github.com/wordpress-mobile/WordPress-Android/pull/16439]
```

Comment:
Present in those WordPress Release Notes, but removed from Jetpack ones — as it's about self-hosted sites which is not applicable to the Jetpack app. \cc @leandroalonso